posthog/models/property/property.py

```diff
@@ -257,6 +257,17 @@ def _parse_value(self, value: ValueT) -> Any:
             return False
         if isinstance(value, int):
             return value
+
+        # `json.loads()` converts strings to numbers if possible
+        # and we don't want this behavior, as if we wanted a number
+        # we would have passed it as a number
+        try:
+            # tests if string is a number & returns string if it is a number
+            int(value)
```

Comment:
Reason to use this vs `isdigit`?

Reply:
1. I didn't know `isdigit` was a thing 😂 
2. That _seems_ to only work on [0-9], vs any random number passed in as a string?

Reply:
oh wait, I was looking at C++ `isdigit`. So (2) isn't an issue.

What's the difference between them?

Reply:
i suppose nothing. i just find isdigit to be a clearer test of what's going on 😶